### ISSUE TITLE
EPUB GEM is pinned at 1.5 and thus incompatible with AsciidoctorJ 2.0

### ISSUE THREAD

**Issue by ysb33r** [asciidoctor/asciidoctorj-epub3#1]
Typical error when running

```
Unable to activate asciidoctor-epub3-1.5.0.alpha.8, because asciidoctor-2.0.2 conflicts with asciidoctor (~> 1.5.0)
```

**Comment by slonopotamus** [comment]
This was fixed in asciidoctor-epub3 1.5.0.alpha.9: asciidoctor/asciidoctor-epub3#185

**Comment by slonopotamus** [comment]
@robertpanzer close this, please
